presto-tpch/src/main/java/com/facebook/presto/tpch/statistics/ColumnStatisticsData.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tpch.statistics;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Optional;
+
+import static java.util.Objects.requireNonNull;
+
+public class ColumnStatisticsData
+{
+    private final Optional<Long> uniqueValuesCount;
```

Comment:
Stats API in spi has `getDistinctValuesCount`, so '...Count' instead of 'numberOfDistinctValues'. I'll rename to `distinctValuesCount` in this round, ping me if that won't be good enough. I assume you don't suggest using the acronym as variable name :)